drv/fe310-rtc/src/main.rs

```diff
@@ -36,10 +46,10 @@ fn main() -> ! {
             };
             num_ints += 1;
 
-            sys_log!("Recieved RTC Interrupt number {}", num_ints);
+            sys_log!("RTC Interrupt number {}", num_ints);
             ringbuf_entry!(num_ints);
 
-            sys_irq_control(RTC_INT, true);
+            unsafe { int_ctrl.complete_int(RTC_INT).unwrap_unchecked() };
```

Comment:
Is the unsafe unwrap really necessary?

Reply:
Because it's just a demo, no. I can change that.